crates/matrix-sdk-ui/src/room_list_service/mod.rs

```diff
@@ -151,8 +151,8 @@ impl RoomListService {
                         (StateEventType::RoomMember, "$ME".to_owned()),
                         (StateEventType::RoomName, "".to_owned()),
                         (StateEventType::RoomCanonicalAlias, "".to_owned()),
-                        (StateEventType::RoomAvatar, "".to_owned()),
```

Comment:
Best leave an explanation for this

Reply:
I cannot add a comment in the code, but I can comment here.

Sliding sync already supplies a `response.avatar` value, so we don't need to request the `m.room.avatar` state event.